librarian_server/orm/file.py

```diff
@@ -145,3 +145,68 @@ def delete(
 
         if commit:
             session.commit()
+
+
+class CorruptFile(db.Base):
+    """
+    An ORM object for a file that has been marked as (potentially) corrupt
+    during a check. This will need to be verified and fixed.
+    """
+
+    __tablename__ = "corrupt_files"
+
+    id: int = db.Column(db.Integer, primary_key=True)
+    "The ID of the corrupt file."
+    file_name: str = db.Column(
```

Comment:
If you maintain the entry the file name can be the primary key here as well. The above code will not create a new entry for a file if it has been corrupted at least once.

Also, I would prefer if this was just `name` it is in sync with the File ORM.